flowcraft/generator/templates/seq_typing.nf

```diff
@@ -34,7 +34,7 @@ process seq_typing_{{ pid }} {
         cp -r /NGStools/ReMatCh rematch_temp
         export PATH="\$(pwd)/rematch_temp/ReMatCh:\$PATH"
 
-        seq_typing.py -f ${fastq_pair[0]} ${fastq_pair[1]} -r \$(pwd)/$refO \$(pwd)/$refH -o ./ -j $task.cpus --extraSeq 0 --mapRefTogether --minGeneCoverage 60
+        seq_typing.py reads -f ${fastq_pair[0]} ${fastq_pair[1]} -r \$(pwd)/$refO \$(pwd)/$refH -o ./ -j $task.cpus --extraSeq 0 --minGeneCoverage 60
```

Comment:
Is this `seq_typing.py` some script you had before? You could use it as a nf template rather than running it directly in the shell. But I guess it is fine if you had this script from previous python versions.

Reply:
Ah, so this seq_typing.py is within the docker image? I expected it to be within the flowcraft python templates but that is fine. Ok.

Reply:
The `seq_typing.py` is not a single script. Maybe the `.py` raised some confusion. The pipeline is inside a Docker image

Reply:
Now I am confused. Is `seq_typing.py` a pipeline?

Reply:
Seq_typing is a pipeline, available here: https://github.com/B-UMMI/seq_typing

Reply:
or rather, a standalone tool

Reply:
To me it doesn't make sense to have this monolites in a modular framework. I didn't go through all the code but I saw some subprocesses being run. I also noticed that this runs Rematch. This defeats the whole purpose of flowcraft. You should take advantage of the modularity already present and/or divide this monolite script into modular components. I believe that you can already re-use some of the existing docker images and modules for some parts of it. Like bowtie2/samtools and so on. Maybe some work is required but it would be much more flexible if you do it like that. Putting a pipeline inside a pipeline builder doesn't make sense to me.

Reply:
I understand what you are saying, but I don't know what to say to you